src/opm/parser/eclipse/EclipseState/Schedule/Network/ExtNetwork.cpp

```diff
@@ -147,6 +158,26 @@ void ExtNetwork::add_node(Node node)
     this->m_nodes.insert({ name, std::move(node) });
 }
 
+void ExtNetwork::add_indexed_node_name(std::string name)
+{
+    this->insert_indexed_node_names.emplace_back(name);
+}
+
+bool ExtNetwork::has_indexed_node_name(std::string name) const
+{
+    // Find given element in vector
+    auto it = std::find(this->insert_indexed_node_names.begin(), this->insert_indexed_node_names.end(), name);
+
+    if (it != this->insert_indexed_node_names.end())
```

Comment:
If you want to save a couple of bytes of source code this could be:
```
return (it != this->insert_indexed_node_names.end());
```

Reply:
OK - done!